File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/C. Even, Odd, Positive and Negative/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/C

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemC
    {
        static void Main(string[] args)
        {
            ushort n = ushort.Parse(Console.ReadLine());
            int even=0, odd=0, pos=0, neg=0;
            string[] input = Console.ReadLine().Split(" ");
            while(n > 0){
                int x = int.Parse(input[--n]);
                if((x&1) == 1) ++odd;
                else ++even;

                if(x > 0) ++pos;
                else if(x < 0) ++neg;
            }
            Console.WriteLine($"Even: {even}");
            Console.WriteLine($"Odd: {odd}");
            Console.WriteLine($"Positive: {pos}");
            Console.WriteLine($"Negative: {neg}");
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/C. Even, Odd, Positive and Negative/main.dart
import 'dart:io';

void main() {
  int n = int.parse(stdin.readLineSync()!);
  int even = 0, odd = 0, pos = 0, neg = 0;
  List<String> input = stdin.readLineSync()!.split(" ");

  while (n > 0) {
    int x = int.parse(input[--n]);
    if ((x & 1) == 1)
      ++odd;
    else
      ++even;

    if (x > 0)
      ++pos;
    else if (x < 0) ++neg;
  }
  print("Even: ${even}");
  print("Odd: ${odd}");
  print("Positive: ${pos}");
  print("Negative: ${neg}");
}
